ext/reflection/php_reflection.stub.php

```diff
@@ -683,3 +687,36 @@ private function __clone(): void {}
 
     private function __construct() {}
 }
+
+final class ReflectionEnum extends ReflectionClass
+{
+    public function __construct(object|string $objectOrClass) {}
+
+    public function hasCase(string $name): bool {}
+
+    public function getCase(string $name): ReflectionEnumUnitCase|false {}
```

Comment:
The RFC says:
```
  // Returns a single reflection object for the corresponding case.
  // If not found, throws, ReflectionException.
  public function getCase(string $name): ReflectionEnumPureCase|ReflectionEnumBackedCase
```

The signature is clearly out of date, but it says that this is supposed to throw rather than return false. I'd be okay with throwing or returning null here, but returning false is the worst possible thing we could do.

Reply:
Good catch. Probably a copy/paste error.